central/cve/telemetry/vulnerabilities.go

```diff
@@ -0,0 +1,167 @@
+package telemetry
+
+import (
+	"context"
+	"strconv"
+
+	"github.com/stackrox/rox/generated/storage"
+	"github.com/stackrox/rox/pkg/search"
+)
+
+// Label is an alias because the central/metrics package cannot import it.
+type Label = string
```

Comment:
Why not using the prometheus type for label?

Reply:
Which one? I see only `type Labels map[string]string`.